data/compatible_products.yml

```diff
@@ -85,6 +85,18 @@ sets:
             - version: 6.4.0
               compatibility: 'https://docs.payara.fish/enterprise/docs/Jakarta%20EE%20Certification/6.4.0/6.4.0%20Platform%20TCK%20Results.html'
               download_url: 'https://www.payara.fish/page/payara-enterprise-downloads/'
+        - name: JBoss Enterprise Application Platform
+          vendor: Red Hat
+          image: '/images/compatible_products/red-hat-jboss-logo.png'
+          image_width: 195
+          download: 'http://www.redhat.com/en/technologies/jboss-middleware/application-platform'
+          versions:
+            - version: 8.0.0, Java 17
+              compatibility: https://github.com/RedHatOfficial/jbosseap-certifications/blob/EE10/EAP-8.0.0/jakarta-core-jdk17.adoc#tck-results
```

Comment:
Please change to refer to https://github.com/RedHatOfficial/jbosseap-certifications/blob/EE10/EAP-8.0.0/jakarta-full-platform-jdk17.adoc